resources/scripts/build_standard_library.py

```diff
@@ -187,13 +250,24 @@ def replace_lang_version(version):
     os.replace(TEMP_PROPERTIES, GRADLE_PROPERTIES)
 
 
-def replace_stdlibs_version():
+def replace_stdlibs_version(module_name, snapshots_build):
+    global version_dict
     with open(TEMP_PROPERTIES, "w+") as temp, open(GRADLE_PROPERTIES) as properties:
         for line in properties:
-            if re.match("^stdlib.*Version=", line):
-                version_string = line.split("=")[0]
-                version_number = line.split("=")[1].split("-")[0]
-                line = version_string + "=" + version_number + "-SNAPSHOT" + "\n"
```

Comment:
By keeping it as 'stdlibConstraintVersion', we can directly access the version number from the dictionary using the 'version_string'.
```
line = version_string + "=" + version_dict[version_string]  + "\n"
```

Reply:
Fixed with 33d788f226ccf744b0fe702ece2da53c1259f4f7